packages/ui/src/components/vuestic-components/va-icon/VaIcon.vdemo.vue

```diff
@@ -1,25 +1,25 @@
 <template>
   <VbDemo>
     <VbCard title="MD">
-      <va-icon class="material-icons">
+      <i class="material-icons">
```

Comment:
Please use `va-icon` instead of `i` we're comparing direct access to config access.